test_mavros/CMakeLists.txt

```diff
@@ -93,25 +100,44 @@ catkin_package(
 ## Your package locations should be listed before other locations
 # include_directories(include)
 include_directories(
+  include
+  ${Boost_INCLUDE_DIRS}
   ${catkin_INCLUDE_DIRS}
 )
 
 ## Declare a cpp library
-# add_library(test_mavros
-#   src/${PROJECT_NAME}/test_mavros.cpp
-# )
+add_library(
+  sitl_test
+  src/sitl_test/px4/lib/sitl_test.cpp
+  include/tests/offboard_posctl_square.h
+  include/sitl_test/sitl_test.h
+  include/sitl_test/test_type.h
+)
 
 ## Declare a cpp executable
-# add_executable(test_mavros_node src/test_mavros_node.cpp)
+add_executable(
+  sitl_test_node
+  src/sitl_test/px4/sitl_test_node.cpp
```

Comment:
Well this was just to separate src code from the rest. But it's just a folder, so we can re-adapt no problem.